### ISSUE TITLE
Hidden scrollbar - Chrome

### ISSUE THREAD

**Issue by blurkness** [getgrav/grav-plugin-admin#659]
I was trying to retest the issue #618 using a clean install but i have some issues with the scrollbar and to update the antimatter theme #658 

I guess it's the #644 is causing the issue with the scrollbar.

The computed CSS:
`body {
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
    overflow: hidden;
}
#admin-main .content-wrapper {
    position: relative;
    overflow-y: hidden;
    height: calc(100vh - 4.2rem);
}`

If i remove the overflow hidden the scrollbar appears again.
I'm using Ubuntu 16.04 and Chrome 53.0.

Thanks!

**Comment by flaviocopes** [comment]
Yep I can see the problem with Canary. If you scroll back to the top and go down again, it appears.

**Comment by flaviocopes** [comment]
Actually it now works for me. Might just be some Canary issue

**Comment by blurkness** [comment]
@flaviocopes even with Windows 10 and Chrome 51.0 i have the same issue. I tried in incognito mode with no extensions enabled to make sure no ext is causing this.

**Comment by dhaupin** [comment]
I also had this issue, although it only appears on a tertiary install, in another account. It was confusing at first until grep for the JS appending `gm-scrollbar-container` class to the `content-wrapper`.  I see the rule (or lack thereof) that is causing it, and it's caused by Cloudflare RocketLoader.

Site1 - original install in grav accnt -> gemini scrollbars work
 - Element: `<div class="content-wrapper gm-scrollbar-container" style="overflow: auto;">`
- Called via: `<script src="/user/plugins/admin/themes/grav/js/admin.min.js?b009f6e0ee" type="text/javascript" defer=""></script>`

Site2 - second install in another accnt -> scrollbars don't work because of RocketLoader
 - Element: `<div class="content-wrapper">`
 - Called via: `<script data-rocketsrc="/user/plugins/admin/themes/grav/js/admin.min.js?f95445e840" type="text/rocketscript" defer="" data-rocketoptimized="true"></script>`

So if anyone else is having this issue, and youre on Cloudflare, try turning off RocketLoader. Hope that helps!
